Makefile

```diff
@@ -4,25 +4,25 @@ SHELL := /bin/bash
 OPERATOR_NAME := machine-deletion-remediation
 
 ## Tool versions
-# OPERATOR_SDK versions at github.com/operator-framework/operator-sdk/releases
-OPERATOR_SDK_VERSION ?= v1.30.0
+# OPERATOR_SDK versions at https://github.com/operator-framework/operator-sdk/releases
+OPERATOR_SDK_VERSION ?= v1.32.0
```

Comment:
any reason for not using 1.33?

Reply:
I'd like to go through the breaking changes after the release

Reply:
There are breaking changes? Didn't notice that... 🙈 😄 But ok of course, just wanted to be sure that it's on purpose